### ISSUE TITLE
ssmanager Does Not Read "-s" Option on v1.13.2

### ISSUE THREAD

**Issue by RebelliousWhiz** [shadowsocks/shadowsocks-rust#767]
![image](https://user-images.githubusercontent.com/15100865/153791420-1de80cc2-e30a-489e-8bf0-11a18d77c17b.png)

I use -s option to define listen interface. Was it expected?

**Comment by zonyitoo** [comment]
What do you expect this `-s` option for?

**Comment by RebelliousWhiz** [comment]
To set ssmanager on a specific interface only, so ssserver can be started on a specific interface only, therefore I can manipulate usages for different interfaces.

Let's say I have a few interfaces, eno0, eno1, eno2.

eno0 for general uses, eno1 for netflix, eno2 for private purposes.

With "-s" option: I can start 3 ssmanager in the background, and control them individually with shadowsocks-manager, so some users can use eno0, some can use eno1, and only I can use eno2. Each port on each interface can be assigned to one user without interference with others.

Without "-s" option: Only 1 ssmanager can be started, ssserver will be boot-up simultaneously on the same port of all interfaces, so users can access eno0, eno1, and eno2 without control, which is not very favorable.

**Comment by RebelliousWhiz** [comment]
Sorry for being confusing, forget about eno0 and eno1. I want to start ssmanager on a specific IP, so ssmanager will only create ssserver base on that specific IP, without creating ssserver on all interfaces (0.0.0.0).

I am familiar with `--outbound-bind-interface`, but `--outbound-bind-interface` allows you to control "outbound interface" only, and it does not control "inbound interface".

Expected behavior:

```
ssmanager -s 192.168.100.100
ssmanager[4489]:  INFO  shadowsocks tcp server listening on 192.168.100.100:15627
```

Current behavior:

```
ssmanager (no -s option)
ssmanager[4489]: INFO  shadowsocks tcp server listening on 0.0.0.0:15627
```

With "-s", I can control which IP users can connect to, without allowing them to connect to any IPs.

"-s" option was supported on v1.12.5, I noticed this because I upgraded to v1.13.2 today and got tons of errors and caused me some pain, had to remove "-s" option for the temporary solution 😂
